components/Cards/index.js

```diff
@@ -17,3 +17,69 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+
+
+// Making API request
+axios.get("https://lambda-times-backend.herokuapp.com/articles")
+
+
+// .then(response => console.log(response.data))
+
+.then(response=>{
+
+    const allArticlesTypes = Object.keys(response.data.articles);
+
+    for(let i=0; i< allArticlesTypes.length; i++){
+
+        response.data.articles[allArticlesTypes[i]].map((articleType)=>{
+
+            const cardContainer = document.querySelector('.cards-container')
+            const createdCard = cardCreator(articleType);
+            cardContainer.appendChild(createdCard);
+            
+        })
+    }
+})
```

Comment:
It is considered best practice to always chain a .catch function to promises in order to handle errors in case the promise is rejected